### DISCUSSION TITLE
A way for fetcher.load to not participate in revalidation

### DISCUSSION THREAD

**Discussion by piotrkulpinski** [remix-run/remix#5272] [Q&A]
With the release of v1.10, there's a note in the changelog:
> fetcher.load calls now participate in revalidation - they always should have to avoid stale data on your page!

Is there a way to disable that? 
I only want my fetcher to load once, when I manually trigger it.

**Comment by piotrkulpinski** [comment]
The main issue is that I add some custom search params on the client side before I do the `fetcher.load` call and once it's revalidated, the params are not used anymore so it returns a wrong response.

**Reply by kiliman** [reply]
I agree. Since `fetcher.load` is an imperative call, you should be able to opt out of revalidation at the call-site, instead of being forced to use `shouldRevalidate`.
